lib/Doctrine/Common/Cache/FileCache.php

```diff
@@ -51,14 +51,14 @@
      */
     public function __construct($directory, $extension = null)
     {
-        if ( ! is_dir($directory) && ! @mkdir($directory, 0777, true)) {
+        if (! is_dir($directory) && ! @mkdir($directory, 0777, true)) {
```

Comment:
We actually use the `if ( ! $foo` syntax in doctrine, that's a desired deviation from PSR-2